go/src/koding/klient/info/publicip/publicip.go

```diff
@@ -64,15 +63,15 @@ func (d *director) EndpointPublicIP() string {
 		return "http://echoip.net"
 	}
 
-	return config.Builtin.Endpoints.URL("ip", protocol.Environment)
+	return config.Builtin.Endpoints.IP
 }
 
 func (d *director) EndpointIsReachable(port string) string {
 	if d.fallback {
 		return "http://rjk.io/test/" + port
 	}
 
-	return config.Builtin.Endpoints.URL("ipcheck", protocol.Environment) + "/" + port
+	return config.Builtin.Endpoints.IPCheck + "/" + port
```

Comment:
this string interpolation can go into config as well and we can get rid of this helper function